data/avgi/avgi intro.txt

```diff
@@ -0,0 +1,867 @@
+			choice
+				`	"What are they saying?"`
+				`	(Remain silent.)`
+			`	Hriso turns to you. "Consideration. I suppose you might be curious about what we are discussing. The local mining guild here says they cannot sustain their operations with the low price on iron mandated by the Consonance, as we are not buying enough rarer metals like titanium to make up for it."`
+			`	"That's stupid," says Sora. "Without cheap steel we wouldn't be able to keep expanding Peripheria or build the Aktina Cylinder."`
+			`	Hriso doesn't translate that, returning to conversing with the miners. Sora pulls out a translator of her own, listening intently to the conversation, but Darius pulls you aside. "I had a talk with a few people on the way here, and they say there's someone who knows more about the first days after Incursion. Come on."`
+			choice
+				`	"What did they say?"`
+			`	"Apparently it's someone we have to meet in person" he grumbles. "I don't like it, but I don't think it's anything dangerous. Just inconvenient." You excuse yourself and walk along with Darius.`
+			`	Before long, you find yourself in some kind of mess hall, dozens of Avgi sipping from colorful drinks. Darius freezes as he spots a short man seated on a crude stool, the only human-style seating in the room, together with a few Avgi.`
+			`	"Darius. I didn't expect to see you here," he says. "And, what's this, another human here in Avgi space?"`
+			`	Darius takes a while to respond. "Leon. It's been a while."`
+			`	Leon snorts. "That's to say the least. Is Sora with you, or did you crawl here by yourself?"`
+			choice
+				`	"She's helping with some negotiations."`
+					goto you
+				`	(Let Darius do the talking.)`
+			`	"She's with someone from the Consonance, negotiating with the miners here," says Darius.`
+			label you
+			`	"And who's this?" Leon says, looking at you. "How'd you end up here, of all places?"`
+			choice
+				`	"Captain <last>. I was stranded here, just like you."`
+				`	"<first>. Pleased to meet you."`
+			`	You quickly explain how you found yourself in Avgi space, and Leon nods. "Figures. And, let me guess, you want to get out, and this Jump Drive of yours won't let you?"`
+			choice
+				`	"Yes."`
+			`	"That explains the questions the good Captain was asking," he says, nodding at Darius. "I've spent years collecting this information, from hearing the official narrative to dredging the corners of Avgi space for rumors. So, before I tell you this, one condition. I've spent months looking for a way to get a ship to take me to... well, you'll see. But not a lot of captains are willing to take their ships out of the Tangled Shroud and through the Aberrant blockade, so if this leads anywhere, I want a bunk on your ship."`
+			choice
+				`	"Agreed."`
+					goto agreed
+				`	"No deal."`
+			`	"Then I'm not telling you anything." He gets up and leaves, carrying his stool with him.`
+				decline
```

Comment:
I don't think `decline` does anything here, since you're in the `on complete` of the conversation. You might want to move this conversation to the `on offer` of the next mission, and have that mission offer on landing.